library/Booster/JsonRpc/Utils.hs

```diff
@@ -260,3 +264,21 @@ diffBy def pat1 pat2 =
             renderBS
             . runExcept
             . internaliseTermOrPredicate DisallowAlias IgnoreSubsorts Nothing def
+
+-------------------------------------------------------------------
+-- logging helpers, adapted from monad-logger-aeson
+
+handleOutput ::
+    (Log.LogLevel -> IO.Handle) ->
+    Log.Loc ->
+    Log.LogSource ->
+    Log.LogLevel ->
+    Log.LogStr ->
+    IO ()
+handleOutput levelToHandle loc src level msg =
+    let bytes = Log.fromLogStr $ Log.defaultLogStr loc src level msg
+     in BS8.hPutStrLn (levelToHandle level) bytes
+
+-- | Run a logging computation, redirecting various levels to the handles specified by the first arguments
+runHandleLoggingT :: (Log.LogLevel -> IO.Handle) -> Log.LoggingT m a -> m a
+runHandleLoggingT = flip Log.runLoggingT . handleOutput
```

Comment:
A bit puzzled that these function live in the `JsonRpc` space... I would have thought `Booster.Util`  is the most logical place for them :man_shrugging: